packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -204,7 +213,7 @@ public void onResponse(Call call, Response response) throws IOException {
     });
   }
 
-  private static JSONObject extractManifest(String manifestString, UpdatesConfiguration configuration) throws IOException {
+  private static JSONObject extractUpdateResponseJson(String manifestString, UpdatesConfiguration configuration) throws IOException {
```

Comment:
This function seems like it is dead code. It is unable to handle the array case in the case that the manifest is signed.

Is there a use case for arrays of `sdkVersions` only in the case of unsigned manifests?

Otherwise we could just replace this with a call to `JSONObject`.

Reply:
> Is there a use case for arrays of `sdkVersions` only in the case of unsigned manifests?

Yes, exactly -- the "array" case is used for multi-manifests generated by `expo export`, which are never signed. While we could add the ability to handle signatures in the body, I think it's better to just focus on supporting the modern method of header signatures moving forward.